2020-09-16 03:13:22,873 INFO    MainThread:9531 [wandb_init.py:_log_setup():307] Logging user logs to wandb/run-20200916_031322-1lig9z3t/logs/debug.log
2020-09-16 03:13:22,873 INFO    MainThread:9531 [wandb_init.py:_log_setup():308] Logging internal logs to wandb/run-20200916_031322-1lig9z3t/logs/debug-internal.log
2020-09-16 03:13:22,873 INFO    MainThread:9531 [wandb_setup.py:_flush():68] setting env: {}
2020-09-16 03:13:22,873 INFO    MainThread:9531 [wandb_setup.py:_flush():68] multiprocessing start_methods=fork,spawn,forkserver
2020-09-16 03:13:22,873 WARNING MainThread:9531 [wandb_setup.py:_flush():68] Ignoring settings passed to wandb.setup() which has already been configured.
2020-09-16 03:13:22,874 INFO    MainThread:9531 [wandb_setup.py:_flush():68] configuring jupyter hooks <wandb.sdk.wandb_init._WandbInit object at 0x7fc8576ace10>
2020-09-16 03:13:23,533 WARNING MainThread:9531 [wandb_setup.py:__init__():214] Ignoring settings passed to wandb.setup() which has already been configured.
2020-09-16 03:13:29,128 INFO    MainThread:9531 [wandb_run.py:_console_start():1103] atexit reg
2020-09-16 03:13:29,129 INFO    MainThread:9531 [wandb_run.py:_redirect():978] redirect: SettingsConsole.WRAP
2020-09-16 03:13:29,130 INFO    MainThread:9531 [wandb_run.py:_redirect():1009] Wrapping output streams.
2020-09-16 03:13:29,130 INFO    MainThread:9531 [wandb_run.py:_redirect():1025] Redirects installed.
2020-09-16 03:13:29,136 INFO    MainThread:9531 [wandb_init.py:_pause_backend():229] pausing backend
2020-09-16 03:13:40,718 INFO    MainThread:9531 [wandb_init.py:_resume_backend():234] resuming backend
2020-09-16 03:13:40,721 INFO    MainThread:9531 [wandb_init.py:_pause_backend():229] pausing backend
2020-09-16 04:03:00,068 INFO    MainThread:9531 [wandb_run.py:_atexit_cleanup():1081] got exitcode: 0
2020-09-16 04:03:00,070 INFO    MainThread:9531 [wandb_run.py:_restore():1053] restore
2020-09-16 04:03:00,076 INFO    MainThread:9531 [wandb_run.py:_wait_for_finish():1185] got exit ret: uuid: "fc5d149f8a944a66872cdae379fc92b2"
response {
  poll_exit_response {
    file_counts {
      wandb_count: 1
    }
    pusher_stats {
      uploaded_bytes: 836
      total_bytes: 836
    }
  }
}

2020-09-16 04:03:02,094 INFO    MainThread:9531 [wandb_run.py:_wait_for_finish():1185] got exit ret: uuid: "2f09644054774db691b44d25284f0c84"
response {
  poll_exit_response {
    done: true
    exit_result {
    }
    file_counts {
      wandb_count: 4
    }
    pusher_stats {
      uploaded_bytes: 3182
      total_bytes: 3182
    }
  }
}

2020-09-16 04:03:03,812 INFO    MainThread:9531 [wandb_run.py:_show_files():1332] logging synced files
2020-09-16 04:03:03,814 INFO    MainThread:9531 [internal.py:handle_exit():137] Internal process exited
